config/config.go

```diff
@@ -41,9 +42,8 @@ func New() *Config {
 // ServerConfig holds a server configuration that is specific to either the
 // DHCPv6 server or the DHCPv4 server.
 type ServerConfig struct {
-	Listener  *net.UDPAddr
-	Interface string
```

Comment:
as we were discussing on Slack, we may want to retain the interface in the config file. Since we don't have a final proposal for the new config format yet, shall we leave it here for now?

Reply:
I've re-added a shim to understand the interface keyword in the configuration file, to mean the same thing as before